Stanza_token/stanza_tokenization.py

```diff
@@ -0,0 +1,100 @@
+import stanza
+import langcodes
+import pandas as pd
+from datasets import load_dataset
+from tqdm import tqdm
+tqdm.pandas()
+pd.set_option("display.max_rows", 1000)
+pd.set_option("display.max_colwidth", None)
+
+dataset = load_dataset("CohereForAI/aya_dataset")
+df = pd.DataFrame(dataset["train"])
+df.drop_duplicates(inplace=True)
```

Comment:
I would not perform this step in this script, but it is a great finding (thanks 😄 )